### PR TITLE
ADDED BASO FINANCE (BASE) TO DEX.TRADES

### PR DESCRIPTION
**Pull request by sophie-analytics**
# Thank you for contributing to Spellbook!

Thank you for taking the time to submit code in Spellbook. A few things to consider:

- If you are a first-time contributor, please sign the CLA by copy & pasting **_exactly_** what the bot mentions in PR comment
- Refer to [docs](#spellbook-contribution-docs) section below to answer questions
- Dune team will review submitted PRs as soon as possible

## Spellbook sub-projects

To prepare for Spellbook to scale in size, Dune team has started to pilot sub-projects:

- For a quick overview, see [the GH discussion on the topic](https://github.com/duneanalytics/spellbook/discussions/5238)
  - Please ask questions and continue the conversation there
- The first spell lineage to move: `tokens.erc20` and all upstream spells which help build it
  - **_Note_**: Please navigate to [tokens](/tokens) root directory to find tokens objects

## Spellbook contribution docs

The [docs](docs) directory has been implemented to answer as many questions as possible. Please take the time to reference each `.md` file within this directory to understand how to efficiently contribute & why the repo is designed as it is 🪄

Example questions to be answered:

- What does each property in the [model config block](/docs/models/model_config_block.md) mean?
- What is the [CI test](/docs/ci_test/ci_test_overview.md) attached to PRs and how can I best utilize it?
- What are the Spellbook [best practices](/docs/general/best_practices.md)?

Please navigate through the [docs](/docs) directory to find as much info as you can.

**Note**: happy to take PRs to improve the docs, let us know 🤝

### CODE REVIEW

File: models/_sector/dex/trades/base/_schema.yml [hunk 1]
```diff
@@ -766,3 +766,22 @@ models:
           seed_file: ref('gridex_base_base_trades_seed')
           filter:
             version: 1
+
+  - name: baso_finance_base_base_trades
+    meta:
+      blockchain: base
+      sector: dex
+      project: baso_finance
```
Comment (Hosuke) [inline models/_sector/dex/trades/base/_schema.yml:774]: Hi @sophie-analytics,
have you added `baso_finance` into `dex.info`?
https://github.com/duneanalytics/spellbook/blob/main/models/dex/dex_info.sql

Also please check for other similar dex.trades pr as well.

**Review by Hosuke — CHANGES_REQUESTED** [review]

File: models/_sector/dex/trades/base/_schema.yml (hunk 1 above)
Comment (sophie-analytics) [reply models/_sector/dex/trades/base/_schema.yml:774]: @Hosuke DONE! I have added it now

File: models/_sector/dex/trades/base/_schema.yml [hunk 2]
```diff
@@ -766,3 +766,22 @@ models:
...
+    description: "baso finance base trades"
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - tx_hash
+            - evt_index
+      - check_dex_base_trades_seed:
+          seed_file: ref('baso_finance_base_base_trades_seed')
+          filter:
+            version: 1
```
Comment (Hosuke) [inline models/_sector/dex/trades/base/_schema.yml:787]: The version seems unmatched to `baso_finance_base.base_trades`, please recheck.

(If the seed file only contains one version, we may remove the filter.)

**Review by Hosuke — CHANGES_REQUESTED** [review]

File: models/_sector/dex/trades/base/_schema.yml (hunk 2 above)
Comment (sophie-analytics) [reply models/_sector/dex/trades/base/_schema.yml:787]: Yeah. It's just one version. I have removed the filter

File: seeds/_sector/dex/baso_finance_base_base_trades_seed.csv [hunk 3]
```diff
@@ -0,0 +1,4 @@
+blockchain,project,version,block_date,tx_hash,evt_index,token_bought_address,token_sold_address,block_number,token_bought_amount_raw,token_sold_amount_raw
+base,baso_finance,1,2024-02-29,0xffd4b1a0feccb552677f66dfad9c9513ab099e8861df3c9680a71403fbdc01a8,97,0xd9aaec86b65d86f6a7b5b1b0c42ffa531710b6ca,0x4200000000000000000000000000000000000006,11219567,20130916,5917311547268586
+base,baso_finance,1,2024-02-29,0x77b30ef875f23152d952956180bb110cf87c583c24d8922e605630503c8a9fa0,8,0x4200000000000000000000000000000000000006,0xd9aaec86b65d86f6a7b5b1b0c42ffa531710b6ca,11218505,3002100810131972,10328676
```
Comment (Hosuke) [inline seeds/_sector/dex/baso_finance_base_base_trades_seed.csv:3]: The version you record here is `1`, while in `baso_finance_base.base_trades` is `2`.
https://github.com/duneanalytics/spellbook/pull/5476/files#diff-df7fe6ef229169722ae0a95c5982d333b6a90199dd992bc5a832131a9993d2b1R17

**Review by Hosuke — CHANGES_REQUESTED** [review]

**Comment by Hosuke** [comment]
Hi @sophie-analytics,
please review the comments above and resolve the merge conflicts.

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**